db/seeders/search_test_data_seeder.rb

```diff
@@ -0,0 +1,217 @@
+class SearchTestDataSeeder
+
+  def initialize
+    @show = CTS::Cases::Show.new
+  end
+
+  def run
+    @case_count= 0
+    while @case_count < 200
+      create_case
+      @case_count += 1
+    end
+  end
+
+  def create_case
+    params = HashWithIndifferentAccess.new({
+            type: select_type,
+            name: select_name,
+            email: select_email,
+            postal_address: select_postal_address,
+            requester_type: select_requester_type,
+            received_date: select_received_date,
+            created_at: select_created_at,
+            delivery_method: select_delivery_method,
+            subject: select_subject,
+            message: select_message,
+            flag_for_team: set_flagged,
+            responding_team: select_responding_team,
+            target_state: select_target_state,
+        })
+
+    ActiveRecord::Base.transaction do
+      case_creator = CTS::Cases::Create.new(Rails.logger, params)
+      @case = case_creator.new_case
+      selected_state = params['target_state']
+      if @case.valid?
+        case_creator.call([selected_state], @case)
+        puts "Case created: #{@case.number}"
+        @show.call(@case)
+      else
+        puts "Failed for these params: "
+        ap params
+        puts "with these errors: "
+        puts @case.errors.full_messages
+      end
+    end
+  end
+
+  def select_type
+    types = [
+              "Case::FOI::Standard",
+              "Case::FOI::ComplianceReview",
```

Comment:
Just a note on efficiency.  Every time select_type (and a number of other methods) is called, you are instantiating a number of strings, and then instantiating an array, and it's the same strings and same array every time.  Better to do 
```@names ||= [ 'name 1', name 2'....]```